patches/jsc_fix_arm64_jit_crash.patch

```diff
@@ -0,0 +1,41 @@
+diff -aur target-org/webkit/Source/JavaScriptCore/assembler/ARM64Assembler.h target/webkit/Source/JavaScriptCore/assembler/ARM64Assembler.h
+--- target-org/webkit/Source/JavaScriptCore/assembler/ARM64Assembler.h	2019-06-18 21:49:21.000000000 +0800
++++ target/webkit/Source/JavaScriptCore/assembler/ARM64Assembler.h	2019-06-19 15:14:46.000000000 +0800
+@@ -2863,7 +2863,36 @@
+
+     unsigned debugOffset() { return m_buffer.debugOffset(); }
+
+-#if OS(LINUX) && COMPILER(GCC_COMPATIBLE)
++#if defined(CUSTOMIZE_REACT_NATIVE)
```

Comment:
why do we need this flag? It also does not seem to be set anywhere

Reply:
It is my habit to enclose modifications by ifdef and it will be clear to see the scope we've patched.
It was first introduced from the `getline` patch for API 16.
Feel free to let me know if you prefer to remove this constant or apply this constant to all patches.
I will have another PR then.